fatjdbc/src/test/java/com/qubole/quark/fatjdbc/test/DDLViewTest.java

```
@@ -0,0 +1,220 @@
+  private int getSize(String sqlQuery) throws  SQLException {
+    Connection connection = DriverManager.getConnection("jdbc:quark:fat:", props);
+    ResultSet resultSet = connection.createStatement().executeQuery(sqlQuery);
+    assertThat(resultSet.next()).isEqualTo(true);
+    connection.close();
+    return resultSet.getInt("count(*)");
+  }
+
+  @Test
+  public void testCreateView() throws SQLException, ClassNotFoundException {
+    Class.forName("com.qubole.quark.fatjdbc.QuarkDriver");
+    String sqlQuery = "select count(*) from warehouse as wr where wr.w_warehouse_sq_ft > 100";
+    assertThat(getSize(sqlQuery)).isEqualTo(0);
+
+    String sql1 = "CREATE VIEW (name, description, cost, query, ds_set_id, destination_id, schema_name, table_name) "
+        + "values(\"warehouse_part\", \"Warehouse Partition\", 0, "
+        + "\"select * from canonical.public.warehouse as wr where wr.w_warehouse_sq_ft > 100\", "
+        + "1, 2, \"PUBLIC\", \"WAREHOUSE_PARTITION\")";
+    Connection connection = DriverManager.getConnection("jdbc:quark:fat:", props);
+    connection.createStatement().executeUpdate(sql1);
+    connection.close();
+
+    assertThat(getSize(sqlQuery)).isEqualTo(1);
+  }
+
+  @Test(expected = SQLException.class)
+  public void testCreateWrongParam() throws SQLException, ClassNotFoundException {
+    Class.forName("com.qubole.quark.fatjdbc.QuarkDriver");
+    String sql1 = "CREATE VIEW (name, description, cost, query, ds_set_id, destination_id, schema_name) "
+        + "values(\"warehouse_part\", \"Warehouse Partition\", 0, "
+        + "\"select * from canonical.public.warehouse as wr where wr.w_warehouse_sq_ft > 100\", "
+        + "1, 2, \"PUBLIC\")";
+    Connection connection = DriverManager.getConnection("jdbc:quark:fat:", props);
+    connection.createStatement().executeUpdate(sql1);
+    connection.close();
+  }
+
+  @Test
+  public void testAlterView() throws SQLException, ClassNotFoundException {
```

Need one more negative test on Alter. Write a test 'Alter view ... where name = <some valid name>'. This should throw an error as only alter on id is allowed.